### PR TITLE
Feature/update solo for posm

### PR DESCRIPTION
**Pull request by jdmedlock**
Updated Solo Project Guide to include tier definition for Scrum Masters. In addition, simplified the description for what's require for each role.

### CODE REVIEW

**Review by onetoughcookie226 — COMMENTED** [review]
Looks good. 
Just not sure about the request for backlog on a public website.

**Comment by jdmedlock** [comment]
> image
> 
> should product owner and ui/ux designer swap in the new diagram?

Not yet. We currently don't have a sample project for UI/UX Designers to submit so we are dependent on them having something to submit based on the Tier definition. This is a deficiency that needs to be corrected in the near future.

File: docs/guides/soloproject/soloproject.md
[the remark this answers is not in the document]
Comment (jdmedlock) [reply docs/guides/soloproject/soloproject.md:1]: Thanks for catching this. I've added additional advice in that paragraph. See if it looks okay now.

**Comment by jdmedlock** [comment]
Verbal approval from Cheryl in our Discord Admin channel. Overriding and merging.

**Pull request merged**